src/specs/TEMPLATE.md

```diff
@@ -19,7 +19,7 @@ You can view an example spec [here](#add-link).
    - For each interaction define:
      - All input tokens, including [AztecType](https://github.com/AztecProtocol/aztec-connect-bridges/blob/master/src/aztec/libraries/AztecTypes.sol) info
      - All output tokens, including [AztecType](https://github.com/AztecProtocol/aztec-connect-bridges/blob/master/src/aztec/libraries/AztecTypes.sol) info
-     - All relevant bridge ids (may fill in after deployment)
```

Comment:
SHould be `BridgeAddressId`

Reply:
Addressed in 3441df17d0a6c77fe9510de3014ba8a985689d9f